java/src/main/java/io/cucumber/gherkin/StringUtils.java

```diff
@@ -114,4 +115,21 @@ private static boolean contains(char[] characters, char c) {
         return false;
     }
 
+    public static class TrimmedText {
```

Comment:
As this represents a text with indentation, it would be more appropriately called indented text.